src/main/java/io/confluent/kafka/connect/datagen/DatagenConnectorConfig.java

```diff
@@ -60,10 +63,13 @@ public static ConfigDef conf() {
         .define(KAFKA_TOPIC_CONF, Type.STRING, Importance.HIGH, KAFKA_TOPIC_DOC)
         .define(MAXINTERVAL_CONF, Type.LONG, 500L, Importance.HIGH, MAXINTERVAL_DOC)
         .define(ITERATIONS_CONF, Type.INT, -1, Importance.HIGH, ITERATIONS_DOC)
-        .define(SCHEMA_STRING_CONF, Type.STRING, "", Importance.HIGH, SCHEMA_STRING_DOC)
-        .define(SCHEMA_FILENAME_CONF, Type.STRING, "", Importance.HIGH, SCHEMA_FILENAME_DOC)
+        .define(SCHEMA_STRING_CONF, Type.STRING, "", new SchemaStringValidator(),
+                Importance.HIGH, SCHEMA_STRING_DOC)
```

Comment:
```suggestion
        .define(SCHEMA_STRING_CONF,
            Type.STRING,
            "",
            new SchemaStringValidator(),
            Importance.HIGH,
            SCHEMA_STRING_DOC
        )
```
nit: formatting, same with other definitions below